### PR TITLE
Native Base UI library installed

### PR DESCRIPTION
**Pull request by aserputov**
## Issue This PR Addresses

Fixes: #2730 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Native Base UI library installed for mobile application. 

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/mobile/package.json [hunk 1]
```diff
@@ -12,10 +12,12 @@
   "dependencies": {
     "expo": "44.0.5",
     "expo-status-bar": "1.2.0",
+    "native-base": "^3.3.4",
```
Comment (TueeNguyen) [inline src/mobile/package.json:15]: We should pin dependency

**Review by TueeNguyen — CHANGES_REQUESTED** [review]

**Comment by TueeNguyen** [comment]
Can you explain the changes in `metro.config.js`

**Comment by aserputov** [comment]
@TueeNguyen yes. Here is the link abut the [bug]( https://github.com/teambit/envs/issues/143)

**Comment by AmasiaNalbandian** [comment]
@aserputov Can you let us know what happened? Would be good to know in future if we try to use this again. I don't understand the bug to be honest.

**Comment by aserputov** [comment]
@AmasiaNalbandian [link](https://github.com/Seneca-CDOT/telescope/issues/2730)

**Comment by AmasiaNalbandian** [comment]
The PR is being closed because React Native essentially has too many bugs to try and work around, and we have decided it will hinder our progress to have a UI library for 2.6. There seems to be a collective decision to try another package, and that is the direction we are headed in https://github.com/Seneca-CDOT/telescope/issues/2730 the issue to install a UI package for RN.

**Pull request closed without merging**